src/Traits/ReportUtilityTrait.php

```diff
@@ -83,4 +83,30 @@ public function summaryLastYear(array $data = array())
         return $this->summary( $data );
     }
 
-}
\ No newline at end of file
+    public function summaryLastDay(array $data = array())

```

Comment:
Can you rename this to `summaryYesterday`? I understand you're trying to be consistent but "last day" isn't really intuitive :-)

Reply:
No problem ;-)